from Utils.Exception.BaseException import BaseException


class GenericException(BaseException):
    pass
